packages/p/pinta/package.yml

```diff
@@ -1,32 +1,31 @@
 name       : pinta
-version    : 1.7.1
-homepage   : https://pinta-project.com/pintaproject/pinta/
-release    : 13
+version    : 2.1.1
+homepage   : https://www.pinta-project.com/
+release    : 14
 source     :
-    - https://github.com/PintaProject/Pinta/releases/download/1.7.1/pinta-1.7.1.tar.gz : cdbbbfe241b8fe5f3a1d0b16e73115125e264a9c94d25fce9e2fcf43429efab9
+    - git|https://github.com/PintaProject/Pinta.git : 2.1.1
```

Comment:
Does this use git submodules? That's the only reason we'd really use a git tag IMO. If you do use that please point it to a specific commit instead.

Reply:
I'll use the tarball